### PR TITLE
[READY] Update blog index pages

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
First iteration to: #286 

- [x] add a featured image option and change the layout of the page 
- [x] add featured images to all existing blogs
- [x] optimize images
- [x] make mobile styling better
- [x] wrap the avatar and author nicely on mobile
- [x] improve mobile padding on article
- [x] make helper for blog author
- [x] make the avatar smaller

Not in scope of this PR, follows later in other ones (need to open issues for them) 
- removing color literals and #main id 
- featured post on top of the index
- related posts below a post with featured images
- featured post on the homepage with featured image

### 1
blog-page-1

### 4
blog-page-4

### CODE REVIEW

File: source/blog/index.html.erb [hunk 1]
```diff
@@ -7,34 +7,43 @@ pageable: true
...
+                <%= link_to t("blog.read_more"), article.url, class: "read-more" %>
               </div>
-            </footer>
 
-          </article>
+              <footer class="post-footer">
+                <div class="post-meta">
+                  <%
+                  author = blog_author(article)
+                  author_name = author ? "#{author.firstname} #{author.prefix} #{author.lastname}" : article.data.author
```
Comment (fatboypunk) [inline source/blog/index.html.erb]: Might want to make an helper for this

File: source/stylesheets/_blog.scss [hunk 2]
```diff
@@ -148,14 +105,75 @@
...
-  .post-header {
-    h2 {
-      font-size: 1.6em;
-      font-weight: 600;
+    li:first-child article {
+      padding-top: 0;
+    }
+
+    li {
+      border-bottom: #ebf2f6 1px solid;
```
Comment (fatboypunk) [inline source/stylesheets/_blog.scss]: Yes please :)

**Comment by sn3p** [comment]
Love it! Few thing I noticed:

- Right padding on post body (mobile):
  screen shot 2016-12-02 at 15 22 52
- Author and date doesn't wrap nicely on mobile:
  screen shot 2016-12-02 at 15 32 08
- I think the avatar is a little bit too large. I prefer:
  ```css
  height: 30px;
  margin-right: 0.2em;
  width: 30px;
  ```

File: source/blog/index.html.erb (hunk 1 above)
Comment (RonaldDijkstra) [inline source/blog/index.html.erb]: Yes!

File: source/stylesheets/_blog.scss (hunk 2 above)
Comment (RonaldDijkstra) [inline source/stylesheets/_blog.scss]: I wanted to do that in another PR but guess when I touch something let's make it right.

File: source/blog/index.html.erb [hunk 3]
```diff
@@ -7,34 +7,43 @@ pageable: true
...
+                  %>
+                  <% if author && href = team_avatar_url(author) %>
+                    <%= image_tag href, alt: author_name, class: "avatar" %>
+                  <% end %>
```
Comment (RonaldDijkstra) [inline source/blog/index.html.erb]: 👍

File: source/stylesheets/_blog.scss (hunk 2 above)
Comment (fatboypunk) [inline source/stylesheets/_blog.scss]: nvm :)

Comment (RonaldDijkstra) [inline source/stylesheets/_blog.scss]: Okay

**Comment by RonaldDijkstra** [comment]
blog-defacto-de
